src/SFA.DAS.Commitments.Database/Tables/TransferRequest.sql

```diff
@@ -9,6 +9,7 @@
 	[TransferApprovalActionedByEmployerEmail] NVARCHAR(255),
 	[TransferApprovalActionedOn] DATETIME2,
 
+    [CreatedOn] DATETIME2 NOT NULL DEFAULT GETDATE(), 
```

Comment:
Do we need the precision of datetime2? And, do we even get it by using getdate()?

Reply:
Phil says use DateTime2